server/config/tables.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable no-console */
+import pool from './db';
+
+pool.on('connect', () => {
+  console.log('Connected to PSQL DB');
+});
```

Comment:
Would you explain why you put this instruction??

Reply:
To track the progress of the script when I run it. I want to see it if connecting to db is successful, response from creating or dropping tables, then closing the connection.